blogPosts.js

```diff
@@ -0,0 +1,9 @@
+let posts = {
+    0: {user: 'George', message: 'Alligator &gt Crocodile'},
+    1: {user: 'Asmahan', message: ''},
+    2: {user: 'Maria', message: ''},
+    3: {user: 'Bereket', message: ''},
+    4: {user: 'Elon', message: 'Damn this is better than twitter. Can I buy? &#128176;'},
+};
+
+module.exports = posts;
```

Comment:
Yaay, glad you went ahead and did it like this!! I feel like we're off to a good start.